src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesClientProvider.java

```diff
@@ -87,8 +87,7 @@
     private KubernetesClientProvider() {
     }
 
-    static KubernetesClient createClient(KubernetesCloud cloud) throws NoSuchAlgorithmException,
-            UnrecoverableKeyException, KeyStoreException, IOException, CertificateEncodingException {
+    static KubernetesClient createClient(KubernetesCloud cloud) throws Exception {
```

Comment:
Keep specialised exceptions, I guess this would become `throws NoSuchAlgorithmException,
            UnrecoverableKeyException, KeyStoreException, IOException, CertificateEncodingException, KubernetesAuthException`